cts/cts-cli.in

```diff
@@ -2227,65 +2227,65 @@ class CrmShadowRegressionTest(RegressionTest):
                                 [Test, ValidatingTest],
                                 expected_rc=ExitStatus.ERROR),
             ], setup=delete_shadow_resource_defaults),
-            # Reset shadow instance (overwrite existing shadow file based on active CIB)
-            Test("Reset shadow instance", "crm_shadow --reset {shadow} --batch"),
-            Test("Get active shadow instance's diff (after reset)", "crm_shadow --diff"),
         ]
 
         reset_1_tests = [
-            ValidatingTest("Reset shadow instance",
+            Test("Reset active shadow instance to active CIB",
+                 "crm_shadow --reset {shadow} --batch"),
+            Test("Active shadow instance no different from active CIB after reset",
+                 "crm_shadow --diff"),
+            Test("Active shadow instance differs from active CIB after change",
+                 "crm_shadow --diff",
+                 setup="crm_attribute -n admin_epoch -v 99",
+                 expected_rc=ExitStatus.ERROR),
+
+            ValidatingTest("Reset active shadow instance to active CIB",
                            "crm_shadow --reset {shadow} --batch --output-as=xml"),
-            ValidatingTest("Get active shadow instance's diff (after reset)",
+            ValidatingTest("Active shadow instance no different from active CIB after reset",
                            "crm_shadow --diff --output-as=xml"),
-            # Reset an inactive shadow instance with no active instance
-            Test("Reset shadow instance (no active instance)",
-                 "crm_shadow --reset {shadow} --batch",
-                 env={"CIB_shadow": None}),
+            ValidatingTest("Active shadow instance differs from active CIB after change",
+                           "crm_shadow --diff --output-as=xml",
+                           setup="crm_attribute -n admin_epoch -v 199",
+                           expected_rc=ExitStatus.ERROR),
+
+            make_test_group("Reset shadow instance to active CIB with nonexistent shadow file",
+                            "crm_shadow --reset {shadow} --batch {fmt}",
+                            [Test, ValidatingTest],
+                            setup="crm_shadow --delete {shadow} --force",
+                            expected_rc=ExitStatus.NOSUCH),
+            make_test_group("Force-reset shadow instance to active CIB with nonexistent shadow file",
+                            "crm_shadow --reset {shadow} --batch --force {fmt}",
+                            [Test, ValidatingTest],
+                            setup="crm_shadow --delete {shadow} --force"),
+            Test("Active shadow instance no different from active CIB after force-reset",
```

Comment:
This is testing after two resets (not after each of text and XML), if that matters to you

Reply:
Right, the original shadow is nonexistent, so it's not worth testing the diffs in both cases